RELEASE_NOTES.md

```diff
@@ -6,6 +6,37 @@
 
 Release Procedure is defined in the [CONTRIBUTING](CONTRIBUTING.md#release-procedure) document.
 
+## v4.4.0
+
+Highlights:
+
+- TODO
+
+<!-- TODO: See [CHANGELOG](https://github.com/umee-network/umee/blob/v4.3.0/CHANGELOG.md) for a full list of changes. -->
+
+### Validators
+
+We changed our GitHub release configuration, and now provide binaries in `.tar.gz` tarball.
+
+### Upgrade instructions
+
+- Download latest binary or build from source.
+- Make sure `libwasmvm.$(uname -m).so` is properly linked
+  - Run the binary to make sure it works for you: `umeed --version`
```

Comment:
```suggestion
- Run the binary to make sure it works for you: `umeed version`
```

Reply:
actually the indentation was correct, we want to be a sub bullet point